packages/pdf-generator/src/components/PDFDocument.js

```diff
@@ -38,6 +39,9 @@ const PDFDocument = ({
                         </Text>
                     </View>
                     <View style={appliedStyles.reportNameWrapper}>
+                        <Text style={ appliedStyles.groupName }>
```

Comment:
Should this be conditionally rendered on the presence or absence of `groupName` ?